package.json

```diff
@@ -13,10 +13,10 @@
     "@dnd-kit/core": "^6.0.8",
     "@dnd-kit/modifiers": "^6.0.1",
     "@dnd-kit/sortable": "^7.0.2",
-    "@emotion/cache": "^11.7.1",
+    "@emotion/cache": "^11.10.5",
```

Comment:
I'd approve the PR but I see some unexpected changes (looks like target branch was updated?).

Reply:
Thanks for point that out, I updated a stack with graphite that contained the parent of this PR